### ISSUE TITLE
Modify browser location from server 

### ISSUE THREAD

**Issue by mwx23** [lona-web-org/lona#157]
This is a feature request.

My use-case is that I have a menu to select items from a route `/items/<item_id>`.

I can of course do with `<a href='/items/123' />`. This however results in a page reload.

I am currently using `<button/>`s with a server-side function that just replaces item in the DOM, mostly because I can with `lona` :)

The only issue with that is it would be nice if the server-side could update the location in the browser if the user wishes to bookmark that page.

Basically the ability to send data to the browser `history.replaceState()` API.
